stdlib/tkinter/__init__.pyi

```diff
@@ -500,7 +500,7 @@ class Misc:
     bbox = grid_bbox
     def grid_columnconfigure(
         self,
-        index: _GridIndex,
+        index: _GridIndex | list[str] | list[int] | list[str | int] | tuple[str | int, ...],
```

Comment:
The [manual page for `grid`](https://www.tcl.tk/man/tcl/TkCmd/grid.html) says this:

> Indices may be integers, window  names or  the  keyword `all`. For `all` the options apply to all rows currently occupied by content windows. For a window name, that window  must  be a content window of this container and the optionsapply to all rows currently occupied by  the  container  window.

So because the only acceptable string  is `"all"`, it doesn't make sense to create a list of strings for this.

```suggestion
        index: _GridIndex | list[int] |tuple[int, ...],
```

Or do you have some use case where you actually want a list of strings?

Apparently these also accept widgets. (Tkinter widgets are wrappers for Tcl's window names, so you can just imagine "widget" when it says "window name".) We don't have to do it in this PR though.